src/components/switch/switch.stories.mdx

```diff
@@ -266,6 +266,27 @@ For more information check our [Validations](?path=/docs/documentation-validatio
   </Story>
 </Preview>
 
+It is possible to use the `tooltipPosition` to override the default placement of tooltips rendered as part of this component.
+
+<Preview>
```

Comment:
Looks like when the tooltip of this example is hovered over the example applies hover state styling too. 
Might be an idea to check the other `group` components just to see if this happens too.

<img width="1385" alt="Screenshot 2021-08-16 at 08 34 22" src="https://user-images.githubusercontent.com/56251247/129527560-402a5953-f0de-4786-aa83-1374728bfd00.png">